sections/transactions.js

```diff
@@ -263,4 +263,22 @@ export default [
     type: "text",
     content: "You can check the property `knex.isTransaction` to see if the current knex instance you are working with is a transaction."
   },
+  {
+    type: "text",
+    content: "Isolation levels can be important for certain invariants you require as part of your business logic. Use it as part of a config object."
```

Comment:
It would be helpful to list all available options for this parameter and mention that it doesn't work for SQLite and Oracle.
Also `Isolation levels can be important for certain invariants you require as part of your business logic.` sounds rather vague, I would suggest rephrasing as something like "In case you need to specify an isolation level for your transaction, you can use a config parameter `isolationLevel`"